src/main/java/se/myhappyplants/client/model/WaterCalculator.java

```diff
@@ -11,44 +11,41 @@ public String calculateWaterLevelToString(String minWater) {
 
         int waterFrequencyInt = Integer.parseInt(minWater);
         if (waterFrequencyInt <= 200) {
-            waterText = "Needs water 4 times a week";
+            waterText = "Needs water once a month";
         }
-        else if (waterFrequencyInt > 200 && waterFrequencyInt <= 400) {
-            waterText = "Needs water 3 times a week";
+        else if (waterFrequencyInt <= 400) {
+            waterText = "Needs water every fortnight";
         }
-        else if (waterFrequencyInt > 400 && waterFrequencyInt <= 600) {
-            waterText = "Needs water 2 times a week";
+        else if (waterFrequencyInt <= 600) {
+            waterText = "Needs water once a week";
         }
-        else if (waterFrequencyInt > 600 && waterFrequencyInt <= 800) {
-            waterText = "Needs water 1 times a week";
-        }
-        else if (waterFrequencyInt > 800) {
-            waterText = "Needs water every other week";
+        else if (waterFrequencyInt <= 800) {
+            waterText = "Needs water once every 5 days";
         }
         else {
-            waterText = "There's no information about the water level";
+            waterText = "Needs water twice a week";
```

Comment:
Kan inte detta bli jätte fel? om vi inte har någon information om en växt som behöver lite vatten så är kanske 2 gånger i veckan för mycket och växten dör? Jag personligen tycker det är smart att ha kvar den "There's no information about the water level" bara för att vara safe? Gör ju inget för projektet i sig men var bara en tanke.

Reply:
Du har rätt, det var inte meningen att ta ersätta en rad, jag bara såg att text-förklaringar var tvärtom till hur mycket vatten behövdes, så jag ändrade ordning på de. Jag fixar!